### PR TITLE
658 updated value for sampling fraction of imaging and scifi bcal

### PR DESCRIPTION
**Pull request by AkshayaVijay**
### Briefly, what does this PR introduce?
Updates the sampling fraction for Imaging and SciFi BCal

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __issue #658 

### Please check if this PR fulfills the following:
- [X] Tests for the changes have been added
- [X] Documentation has been added / updated
- [X] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No
### Does this PR change default behavior?
Yes

### CODE REVIEW

File: src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelScFiRecHits.h [hunk 1]
```diff
@@ -36,7 +36,7 @@ class CalorimeterHit_factory_EcalBarrelScFiRecHits : public eicrecon::JFactoryPo
         m_thresholdValue=0.0;//{this, "thresholdValue", 0.0};
 
         // energy correction with sampling fraction
-        m_sampFrac=0.125;// from ${DETECTOR_PATH}/calibrations/emcal_barrel_calibration.json
+        m_sampFrac=0.10;// from ${DETECTOR_PATH}/calibrations/emcal_barrel_calibration.json
```
Comment (wdconinc) [inline src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelScFiRecHits.h:39]: Can we get some more significant figures here?

File: src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelImagingRecHits.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ class CalorimeterHit_factory_EcalBarrelImagingRecHits : public eicrecon::JFactor
         m_pedSigmaADC=14; // {this, "pedestalSigma", 3.2};
         m_thresholdFactor=3.0; // {this, "thresholdFactor", 3.0};
         // Calibration!
-        m_sampFrac=0.005;// from ${DETECTOR_PATH}/calibrations/emcal_barrel_calibration.json
+        m_sampFrac=0.006;// from ${DETECTOR_PATH}/calibrations/emcal_barrel_calibration.json
```
Comment (wdconinc) [inline src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelImagingRecHits.h:41]: Can we get some more significant figures here?

**Comment by wdconinc** [comment]
Can you post the figures here that this is based on, for record keeping.

**Comment by AkshayaVijay** [comment]
The plots for the simulation of 10000 events to calculate the sampling fraction for Imaging and SciFi BCal for 1GeV, 5GeV, and 10 GeV are attached.
________________________________

**Comment by wdconinc** [comment]
@mariakzurek Are these the only places to update? I think so.

**Pull request merged**